src/deployment/bicep-templates/function-settings-disabled-apps.bicep

```diff
@@ -0,0 +1,92 @@
+param functions_disabled_setting string
+
+var allFunctions = [
+  'agent_can_schedule'    //0
```

Comment:
make sure the to use the PascalCase version of these function for the dotnet code

Reply:
Good catch, fixed this in commit #24c9fd8